docs/trellis/master/bedrock-integration.md

```diff
@@ -0,0 +1,57 @@
+---
+description: Configuring composer HTTP basic authentication for private packages.
+---
+
+# Bedrock Integration
+
+If you want to manage your WordPress project configuration using Composer, we recommend using the [Bedrock Boilerplate](https://roots.io/docs/bedrock/master/installation/). Bedrock's [Composer](https://roots.io/docs/bedrock/master/composer/) support allows you to install plugins and themes from the WordPress.org repositories, via [WPackagist](https://wpackagist.org/).
```

Comment:
Trellis is already designed for Bedrock sites (or at least ones using Composer for dependency management) so anyone using Trellis already manages it with Composer.

Reply:
> but I don't think this should be called "Bedrock integration"

My bad.

Reply:
Thanks, @swalkinshaw I wasn't sure of this myself, I will update the PR.